src/ui/Analytics/LiveAnalyticsClient.ts

```diff
@@ -269,10 +269,26 @@ export class LiveAnalyticsClient implements IAnalyticsClient {
     return this.resolveActiveTabFromElement(element) || 'default';
   }
 
-  private buildMetaObject<TMeta>(meta: TMeta): IChangeableAnalyticsMetaObject {
-    var build: IChangeableAnalyticsMetaObject = _.extend({}, meta);
-    build['JSUIVersion'] = version.lib + ';' + version.product;
-    return build;
+  private buildMetaObject<TMeta>(meta: TMeta, result?: IQueryResult): IChangeableAnalyticsMetaObject {
+    let modifiedMeta: IChangeableAnalyticsMetaObject = _.extend({}, meta);
+    modifiedMeta['JSUIVersion'] = version.lib + ';' + version.product;
+
+    if (result) {
+      let fieldValue;
+      let fieldUsed;
+      let uniqueId = result.raw['uniqueid'];
+      if (uniqueId) {
+        fieldUsed = 'uniqueid';
+        fieldValue = uniqueId;
+      } else {
+        fieldUsed = 'urihash';
+        fieldValue = result.raw['urihash'];
+      }
+      modifiedMeta['contentIdKey'] = fieldUsed;
```

Comment:
On the backend the casing we receive is `contentIDKey` and `contentIDValue` - I expect you should use the same here.